packages/clerk-js/src/ui/contexts/utils.ts

```diff
@@ -13,3 +13,20 @@ export function assertContextExists(contextVal: unknown, providerName: string):
     clerkCoreErrorContextProviderNotFound(providerName);
   }
 }
+
+export function isRedirectForSSOFlow(clerk: Clerk, redirectUrl: string): boolean {
+  const fapiUrl = clerk.publishableKey || clerk.frontendApi;
+
+  const url = new URL(redirectUrl);
+  const path = url.pathname;
+
+  if (url.host === fapiUrl && ssoRedirectPaths.includes(path)) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+const ssoRedirectPaths: string[] = [
```

Comment:
This mechanism of "stealing" the dev browser JWT from ClerkJS and redirecting back to FAPI will be generalized for SSO and other redirection flows.

@alex-ntousias Do you feel the approach of hardcoding FAPI paths in ClerkJS can scale or should we consider introducing a new query string parameter that ClerkJS will identify, and will always redirect to it adding the dev browser JWT?

Reply:
Yes agreed, we should implement this keeping in mind other SSO and redirection flows. I hard coded the `/oauth/authorize` path to have something simple that worked, but this is also my biggest question for this PR. Happy to change this to a new query string param (or some other solution that is more general), depending on what we think is best here.

Reply:
Yes, hardcoding the path will work. We'll probably need to add the following:
* `/v1/verify`: this is what magic links are using currently
* `/v1/tickets/accept`: this is the generic one for the ticket flow...ideally all these redirection flows should go via this endpoint, but this is something we can discuss in the future

Reply:
Added those paths!